### PR TITLE
feat: introduce default torrent node config

### PR DESCRIPTION
**Pull request by PascalPrecht**

### CODE REVIEW

File: src/app_service/service/accounts/service.nim [hunk 1]
```diff
@@ -334,6 +334,12 @@ proc login*(self: Service, account: AccountDto, password: string): string =
         "Enabled": true,
         "OpenseaAPIKey": OPENSEA_API_KEY_RESOLVED
       },
+      "TorrentConfig": {
+        "Enabled": false,
+        "DataDir": joinPath(main_constants.defaultDataDir(), "data", "archivedata"),
+        "TorrentDir": joinPath(main_constants.defaultDataDir(), "data", "torrents"),
+        "Port": 9025
+      }
```
Comment (jrainville) [inline src/app_service/service/accounts/service.nim:342]: Can this one refer to the network_constants, or they can have different values?

Comment (PascalPrecht) [reply src/app_service/service/accounts/service.nim:342]: Can definitely move it to constants. This however provides a way to change it on a per desktop client instance basis.

**Comment by PascalPrecht** [comment]
Reusing networks constant now

**Review by alaibe — APPROVED** [review]

**Review by noeliaSD — APPROVED** [review]
LGTM

**Pull request merged**